source/sop/sop_040_nl_imagej_ongewervelden/06_werkwijze.Rmd

```diff
@@ -0,0 +1,93 @@
+# Werkwijze
+
+```{=html}
+<!--
+Gedetailleerde omschrijving van alle stappen die doorlopen moeten worden om het protocol uit te voeren.
+Deze stappen staan schematisch weergegeven in het processchema.
+Subtitels gebruiken om elke stap te omschrijven.
+-->
+```
+
+## Openen van de foto
+
+Open ImageJ [2]
+
+Laad de groepsfoto in via File \> Open
+
+De groepsfoto’s bevinden zich in de map: `G:\Gedeelde drives\Procesbeheer\PB11\_Plotbemonstering\PB\_Plot_Invertebraten\FOTO\_SubPlot_Invertebraten\Groepsfoto`
```

Comment:
Deze lange foldernaam wordt niet volledig getoond in zowel de html als de pdf versie. Is het nodig om deze foldernaam in het protocol op te nemen? Kan je het algemener schrijven, bv "Sla de groepsfoto's op in een map met de naam Groepsfoto."? 
Indien je dit toch wil opnemen in het protocol, dan zou ik volgende manier gebruiken:

````
```{r results = "asis"}
G:\Gedeelde drives\Procesbeheer
    |-- PB11
        |-- _Plotbemonstering
            |-- PB
                |--__Plot_Invertebraten
                    |-- FOTO
                         |-- _SubPlot_Invertebraten
                             |-- Groepsfoto
```
````